aws/rust-runtime/aws-config/Cargo.toml

```diff
@@ -22,8 +22,8 @@ tcp-connector = ["tokio/net", "tower"]
 # SSO is not supported
 sso = []
 
-rustls = ["aws-smithy-client/rustls"]
-native-tls = ["aws-smithy-client/native-tls"]
+rustls = ["aws-smithy-client/rustls", "aws-sdk-sts/rustls", "aws-hyper/rustls"]
```

Comment:
do we actually use `aws-sdk-sts::new` or `from_conf`? If not, we don't even need these features

Reply:
You were right, I have removed these